### PR TITLE
made changes to navbar, footer

### PR DESCRIPTION
**Pull request by reveurguy**
Signed-off-by: reveurguy <59417546+reveurguy@users.noreply.github.com>

- Added background to hero,footer. 
- made changes to navbar style
- updated fontawesome version

### CODE REVIEW

**Comment by GuptaManan100** [comment]
@reveurguy If you were doing the open-close draft-ready for getting the CI to run, then that is not required. For external contributors, netlify requires approval from a member to run the preview. I just approved it, so they'll run now.

**Review by GuptaManan100 — COMMENTED** [review]
I like the changes but I am not sure why we have removed the `yarn.lock` file. I'll let someone with more web experience have the final word.

**Comment by reveurguy** [comment]
> @reveurguy If you were doing the open-close draft-ready for getting the CI to run, then that is not required. For external contributors, netlify requires approval from a member to run the preview. I just approved it, so they'll run now.

ok, yeah i was doing that for that only😅

**Comment by reveurguy** [comment]
> I like the changes but I am not sure why we have removed the `yarn.lock` file. I'll let someone with more web experience have the final word.

that was updated automatically, when i ran the site locally. it is not deleted but most of it's content was removed.

**Comment by deepthi** [comment]
@reveurguy can you attach before and after screenshots? Alternatively, you can link to the specific changed pages in the preview.
For instance, I can see that you have changed the background on the homepage, but I can't tell what else has changed.

**Comment by deepthi** [comment]
@nate-double-u @chalin can one of you review the changes? the diff on `yarn.lock` is huge.

**Comment by reveurguy** [comment]
# Before

Screenshot 2022-07-28 at 8 57 32 PM

# After
Screenshot 2022-07-28 at 8 57 18 PM

# before 
Screenshot 2022-07-28 at 8 58 49 PM

# after
Screenshot 2022-07-28 at 8 58 34 PM

**Comment by reveurguy** [comment]
initially i have made these small changes only.

**Comment by deepthi** [comment]
@reveurguy can you look at this on mobile? The changes to the button shapes mean that they look kind of awkward now because one of them is rounded on the left, one is a rectangle, and the third one is rounded on the right.

File

**Comment by chalin** [comment]
> @nate-double-u @chalin can one of you review the changes? the diff on `yarn.lock` is huge.

My suggestion is to drop the lock file from version control:

- #1106

**Comment by chalin** [comment]
I'd suggest leaving the footer the way it was. The new styling would make the text difficult to read for some. My 2 cents.

**Comment by chalin** [comment]
@thisisobate - any comments about the suggested changes?

**Comment by nate-double-u** [comment]
> My suggestion is to drop the lock file from version control:

+1

(i also tend to keep the `package-lock.json` file out of version control. I know they're designed to be included, but they always seemed like more trouble than they're worth to me.)

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {{ $year       := now.Year }}
 {{ $footerLogo := "img/logos/vitess-horizontal.png" | relURL }}
 {{ $social     := site.Params.social }}
-<footer class="footer has-background-black">
+<footer class="footer has-background">
   <div class="container">
     <img class="is-footer-logo" src="{{ $footerLogo }}">
 
-    <hr class="has-background-orange">
+    <hr class="has-background-black">
```
Comment (thisisobate) [inline layouts/partials/footer.html]: revert the `hr` color back to orange as the black foreground doesn't match with the background image

File: layouts/partials/footer.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
     <nav class="level">
```
Comment (thisisobate) [inline layouts/partials/footer.html:10]: The Nav color doesn't match well with the background; can barely read the text. I think you should use a text color that has good color contrast and matches the background- white is a good candidate.

Comment (thisisobate) [reply layouts/partials/footer.html:10]: Also, the external link's contrast isn't great. You should increase it (very light blue color might do the trick?)

**Review by thisisobate — CHANGES_REQUESTED** [review]
+1 on the lock file as @nate-double-u and @chalin suggested.

Add a background image to the footer or leave it as it was before? either way works tbh.

@reveurguy If you want to go with the former, then you should fix the readability issues @chalin mentioned.

File: layouts/partials/footer.html (hunk 1 above)
Comment (thisisobate) [reply layouts/partials/footer.html]: Screenshot 2022-07-30 at 04 58 58

The Nav color doesn't match well with the background; can barely read the text. I think you should use a text color that has good color contrast and matches the background- white is a good candidate.

Comment (thisisobate) [reply layouts/partials/footer.html]: Also, the external link's contrast isn't great. You should increase it (very light blue color might do the trick?)

**Comment by reveurguy** [comment]
made the  required changes

**Review by deepthi — APPROVED** [review]
Looks like all reviews have been addressed.
Re `yarn.lock`, more changes are required to drop `yarn` completely but they can be done separately whenever someone picks up #1106 
@reveurguy welcome to Vitess and thank you for the contribution!

**Pull request merged**